translations/fr/try_ruby_190.md

```diff
@@ -0,0 +1,17 @@
+---
+lang:   FR
+title:  Malheuresement, la poésie de biscotte c'est pas ton truc
+answer: (toast){0}
+load:   prev
+ok:     Excellent
+error:  Ça sent toujours la biscotte pour moi
+---
+
+Eh, c'est pas grave. T'es pas obligé d'aimer ça. Laisses toi aller et
+améliore mon poème. Au lieu d'une biscotte, tu peux essayer un melon
```

Comment:
In order to change the poem in french, we should find two feminin or two masculine things so that the poem doesn't look weird when one word is replaced by another.

E.g. `un toast` and `un melon`, or `une biscotte` and `une citrouille`.